blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+
+In the following sample, `Zoom` effect is enabled. So, The Dialog will open with `ZoomIn` and close with `ZoomOut` effects.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-animation.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-animation.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-animation.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+## Position
+
+You can customize the dialog position by using [Position](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.Position.html#fields) property. Use the following code to customize the dialog position.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-position.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-position.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-position.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+![Alert position Dialog](./images/blazor-alert-position.png)
+
+![Confirm position Dialog](./images/blazor-confirm-position.png)
+
+![Prompt position Dialog](./images/blazor-prompt-position.png)
+
+## Dimension
+
+You can customize the dialog dimensions using [DialogDimension.Height](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogDimension.html#Syncfusion_Blazor_Popups_DialogDimension_Height) and [DialogDimension.Width] properties. Use the following code to customize the dialog dimensions.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-dimension.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-dimension.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-dimension.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+![Alert dimension Dialog](./images/blazor-alert-dimension.png)
+
+![confirm dimension Dialog](./images/blazor-confirm-dimension.png)
+
+![prompt dimension Dialog](./images/blazor-prompt-dimension.png)
+
+## Close Button Dialog
+
+You can customize the close icon using [ShowCloseIcon](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialog.html#Syncfusion_Blazor_Popups_SfDialog_ShowCloseIcon) property. If the `ShowCloseIcon` property is set to `true` then the close icon will appear. Use the following code to enable the `ShowCloseIcon`.
```

Comment:
You can customize the close icon using [ShowCloseIcon](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialog.html#Syncfusion_Blazor_Popups_SfDialog_ShowCloseIcon) property. If the `ShowCloseIcon` property is set to `true` then the close icon will appear. Use the following code to enable the `ShowCloseIcon`: